typespec-tests/src/test/java/com/cadl/model/generated/ModelOpPutNestedTests.java

```diff
@@ -17,7 +17,7 @@ public final class ModelOpPutNestedTests extends ModelClientTestBase {
     @Disabled
     public void testModelOpPutNestedTests() {
         // method invocation
-        NestedModel response = modelClient.putNested(null);
+        NestedModel response = modelClient.putNested(new NestedModel(new NestedModel1(new NestedModel2("123"))));
```

Comment:
This is due to a fix [here](https://github.com/Azure/autorest.java/pull/2921/files#diff-ad84edf0aad661e6e721088dc8c894fa8099fc3ce43a39226a22ea79971ea65d)